rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.56"
+components = [ "rustfmt", "clippy" ]
```

Comment:
I'm not sure of the details for rust-toolchain, but we also do `rustup component add rust-std --target wasm32-unknown-unknown` to get the stdlib for wasm. Is that something that should be included here?

Reply:
Yes. In fact, I did include the wasm32 target when I first wrote this file, but removed it after I double-checked the wasm-pack docs to see what their guidance was for installation. Seemed like wasm-pack satisfies the target when installed or invoked.

I'll add it to the file.

Reply:
I'm not 100% sure about the intersection of targets and components. Since this is technically cross-compilation, we might not be able to avoid invoking `rustup` directly for this case now that I think about it.